liquibase-service/src/main/resources/db/rdb_modern/routines/018-sp_lab100_datamart_postprocessing-001.sql

```diff
@@ -0,0 +1,1799 @@
+        lt3.LAB_RPT_LAST_UPDATE_DT,
+        lt3.JURISDICTION_CD,
+        lt3.LAB_TEST_CD_SYS_NM,
+        lt3.JURISDICTION_NM,
+        lt3.OID,
+        lt3.ACCESSION_NBR,
+        lt3.SPECIMEN_SRC,
+        lt3.SPECIMEN_DESC,
+        lt3.SPECIMEN_SITE,
+        lt3.SPECIMEN_SITE_DESC,
+        lt3.SPECIMEN_COLLECTION_DT,
+        lt3.RESULTED_TEST_UID,
+        lt3.ROOT_ORDERED_TEST_PNTR,
+        lt3.PARENT_TEST_PNTR,
+        lt3.TEST_RESULT_GRP_KEY,
+        lt3.PERFORMING_LAB_KEY,
+        lt3.ALT_LAB_TEST_CD,
+        lt3.ALT_LAB_TEST_CD_DESC,
+        lt3.ALT_LAB_TEST_CD_SYS_CD,
+        lt3.ALT_LAB_TEST_CD_SYS_NM,
+        lt3.RESULTED_LAB_TEST_CD_DESC,
+        lt3.RESULTEDTEST_CD_SYS_NM,
+        lt3.RESULT_TEST_METHOD_CD,
+        lt3.RESULTED_LAB_TEST_KEY,
+        lt3.lab_test_type,
+        lt3.RESULT,
+        lt3.TEST_RESULT_VAL_CD,
+        lt3.TEST_RESULT_VAL_CD_SYS_NM,
+        lt3.LOCAL_RESULT_CODE,
+        lt3.LOCAL_RESULT_NAME,
+        lt3.RESULT_REF_RANGE_FRM,
+        lt3.RESULT_REF_RANGE_TO,
+        lt3.RESULTEDTEST_VAL_CD,
+        lt3.RESULTEDTEST_VAL_CD_DESC,
+        lt3.LAB_RESULT_TXT_VAL,
+        lt3.NUMERIC_RESULT_WITHUNITS,
+        lt3.LAB_RESULT_COMMENTS,
+        lt3.ORDERED_TEST_UID,
+        lt3.code_seq,
+        lt3.code_set_nm,
+        lt3.nbs_uid,
+        lt3.prog_area_cd,
+        lt3.prog_area_desc_txt,
+        lt3.PROGRAM_AREA_ID,
+        lt3.status_cd,
+        lt3.status_time,
+        case
+            when lt3.TEST_RESULT_VAL_CD   like '%[^0-9]%' and SUBSTRING(lt3.TEST_RESULT_VAL_CD,2,1) = '-'
+                     and (lt3.CONDITION='' or lt3.CONDITION is null) and lt3.CONDITION_SHORT_NM is null then substring(sc.DISEASE_NM,1,50)
+            else lt3.CONDITION_SHORT_NM
+        end as CONDITION_SHORT_NM,
+        lt3.LOINC,
+        case
+            when TEST_RESULT_VAL_CD   like '%[^0-9]%' and SUBSTRING(TEST_RESULT_VAL_CD,2,1) = '-'
+                     and (CONDITION='' or CONDITION is null) then substring(sc.DISEASE_NM,1,50)
+            else lt3.CONDITION
+        end as CONDITION,
+		case
+		    when lt3.TEST_RESULT_VAL_CD like '%[^0-9]%' and SUBSTRING(lt3.TEST_RESULT_VAL_CD,2,1) = '-' then lt3.TEST_RESULT_VAL_CD
+            else cast(null as [varchar](1000) )
+        end as SNOMED
+	into #TMP_LABTESTS4
+	from #TMP_LABTESTS3 lt3
+	left outer join nbs_srte.dbo.SNOMED_CONDITION sc with(NOLOCK) on sc.SNOMED_CD = lt3.TEST_RESULT_VAL_CD
+	;
+
+
+	SELECT @ROWCOUNT_NO = @@ROWCOUNT;
+	INSERT INTO dbo.[JOB_FLOW_LOG]
+	(BATCH_ID,[DATAFLOW_NAME],[PACKAGE_NAME] ,[STATUS_TYPE],[STEP_NUMBER],[STEP_NAME],[ROW_COUNT])
+	VALUES
+	(@BATCH_ID,'LAB100_DATAMART','LAB100_DATAMART','START',@PROC_STEP_NO,@PROC_STEP_NAME,@ROWCOUNT_NO);
+
+	COMMIT TRANSACTION;
+
+if @debug = 'true' select 'TMP_LABTESTS4', * from #TMP_LABTESTS4;
+
+	BEGIN TRANSACTION;
+
+	SET @PROC_STEP_NO =  @PROC_STEP_NO + 1;
+	SET @PROC_STEP_NAME = ' GENERATING LAB100 Table - Update';
+
+
+	update dbo.LAB100_1
```

Comment:
Is this supposed to be LAB100?

Reply:
added originally to avoid conflict. It can be changed.

Reply:
If its in rdb_modern, unless the post-processing service wires it up, this should be good to keep as the target table.